src/Components/Blueprints/BuildImagesButton.tsx

```diff
@@ -1,20 +1,160 @@
-import React from 'react';
+import React, { useState } from 'react';
 
-import { Button, ButtonProps } from '@patternfly/react-core';
+import {
+  Dropdown,
+  MenuToggle,
+  MenuToggleElement,
+  Menu,
+  MenuContent,
+  MenuList,
+  MenuItem,
+  Flex,
+  FlexItem,
+  Spinner,
+  MenuToggleAction,
+  ButtonProps,
+  Button,
+} from '@patternfly/react-core';
+import { skipToken } from '@reduxjs/toolkit/query';
 
 import { selectSelectedBlueprintId } from '../../store/BlueprintSlice';
 import { useAppSelector } from '../../store/hooks';
-import { useComposeBlueprintMutation } from '../../store/imageBuilderApi';
+import {
+  ImageTypes,
+  useComposeBlueprintMutation,
+  useGetBlueprintQuery,
+} from '../../store/imageBuilderApi';
 
-type BuildImagesButtonPropTypes = {
+export const BuildImagesButton = () => {
+  const selectedBlueprintId = useAppSelector(selectSelectedBlueprintId);
+  const [selectedTargets, setSelectedTargets] = useState<ImageTypes[]>([]);
```

Comment:
Nit pick, but since the logic works on subtraction would it make sense to call this "deselectedTargets"?